Makefile

```diff
@@ -449,9 +449,9 @@ set-version:
 	NEXT_BUILD=$$(($$CURRENT_BUILD + 1)); \
 	/usr/libexec/PlistBuddy -c "Set :CFBundleVersion $$NEXT_BUILD" $(INFO_PLIST)
 
-ios: macos build-framework ffigen
+#ios: macos build-framework ffigen
```

Comment:
Is this supposed to be commented out? There's a step in CI that depends on this to build the Lantern library:

```
      - name: Build Lantern Library
        shell: bash
        run: make ${{matrix.platform}}
        env:
          VERSION: ${{ env.version }}
```

We still need to do `make build-framework`, right?

Reply:
No we don't, build-framework renamed to ios.